### PR TITLE
Add transactions resource

### PR DESCRIPTION
**Pull request by nathanielmiller20**
Seems we don't have some python version binaries we need? About half the versions are passing and about half can't be found.

### CODE REVIEW

File: pybutton/resources/transactions.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    '''
+
+    def all(self, cursor=None, start=None, end=None, time_field=None):
+        '''Get a list of transactions.
+        To paginate transactions, pass the result of response.next_cursor() as
+        the cursor argument.
+
+
+        Args:
+            account_id (str) optional: A Button account id ('acc-XXX')
```
Comment (griffinmyers) [inline pybutton/resources/transactions.py]: Copypasta

File: README.rst [hunk 2]
```diff
@@ -317,6 +318,34 @@ Get
     print(response)
     # <class pybutton.Response id: customer-1234, ...>
 
+Transactions
```
Comment (griffinmyers) [inline README.rst:321]: please add description for how this differs from accounts#transactions.

File: README.rst [hunk 3]
```diff
@@ -317,6 +318,34 @@ Get
...
+
+All
+'''
+
+You may pass the following optional arguments:
+
+* ``cursor`` (string): An API cursor to fetch a specific set of results.
+* ``start`` (ISO-8601 datetime string): Fetch transactions after this time.
+* ``end`` (ISO-8601 datetime string): Fetch transactions before this time.
+* ``time_field`` (string): Which time field ``start`` and ``end`` filter on.
```
Comment (griffinmyers) [inline README.rst:332]: default?

**Review by griffinmyers — CHANGES_REQUESTED** [review]
Looking great! a few comments.

File: README.rst [hunk 4]
```diff
@@ -317,6 +318,34 @@ Get
...
+
+.. code:: python
+
+    from pybutton import Client
+
+    client = Client('sk-XXX')
+
+    response = client.accounts.transactions(
```
Comment (griffinmyers) [inline README.rst]: `client.transactions`?

**Comment by griffinmyers** [comment]
Did you want to add `time_field` to the accounts resource as well?

**Comment by nathanielmiller20** [comment]
`time_field` is already in the accounts resource for this client: https://github.com/button/button-client-python/blob/master/pybutton/resources/accounts.py#L44

File: pybutton/resources/transactions.py (hunk 1 above)
Comment (nathanielmiller20) [reply pybutton/resources/transactions.py]: Good catch! Done.

File: README.rst (hunk 2 above)
Comment (nathanielmiller20) [reply README.rst:321]: Added, not sure if I used the right convention in the description, though.

File: README.rst (hunk 3 above)
Comment (nathanielmiller20) [reply README.rst:332]: Added.

File: README.rst (hunk 4 above)
Comment (nathanielmiller20) [reply README.rst]: Fixed! Thanks!

**Review by griffinmyers — APPROVED** [review]
LGTM! I think you have one linting failure that is a consequence of a branch I merged underneath you--let's make sure the build passes and merge up!

**Pull request merged**